### DISCUSSION TITLE
Manage user packages site with pdm -g

### DISCUSSION THREAD

**Discussion by snowwm** [pdm-project/pdm#663] [Ideas]
AFAIK, `pdm add -g` can only install packages to system python site (so it needs `sudo`).
I think it's cleaner to only install packages there with system package manager (e. g. pacman). I'd like to use `pdm * -g` as a way to manage user site, as with `pip install --user`.
If this isn't possible now, I propose adding a setting for it.

And great big thank you to @frostming for this awesome project!

**Comment by frostming** [comment]
> AFAIK, `pdm add -g` can only install packages to system python site (so it needs sudo).

It depends on which python you choose for the global project, yes, `pdm use -g` to switch it. But anyway, I am fine with the improvement. Thank you.

**Reply by Hnasar** [reply]
Thank you! Just commenting for others who find this that [the configuration option](https://pdm.fming.dev/configuration/) can be enabled with:

```
pdm config global_project.user_site True
```
